subnet_sharing.tf

```diff
@@ -0,0 +1,9 @@
+module "subnet_sharing" {
+  source                = "git@github.com:schubergphilis/terraform-aws-mcaf-subnet-sharing.git?ref=v0.2.0"
+  count                 = var.share_public_subnets || var.share_private_subnets ? 1 : 0
+  name                  = var.name
+  private_subnet_arns   = var.share_private_subnets ? toset(aws_subnet.private[*].arn) : []
+  public_subnet_arns    = var.share_public_subnets ? toset(aws_subnet.public[*].arn) : []
+  prepend_resource_type = var.prepend_resource_type
+  tags                  = var.subnet_sharing_custom_tags
```

Comment:
I thing you should merge the custom tags with the default tags. Otherwise you will miss the default ones or have no tags if this is not set.

Reply:
Since we have a subnet_sharing_custom_tags variable shouldn't the merging be done on the value of that?